### DISCUSSION TITLE
새로고침 주기

### DISCUSSION THREAD

**Discussion by anzpek** [changdoc/naver-vaccine-macro#204] [Q&A]
창2개는 띄우고 싶은데 새로고침 주기를 어떻게 바꾸나요?

**Comment by changdoc** [comment]
![image](https://user-images.githubusercontent.com/1879494/127995961-f7906023-410b-471c-9a9c-799ab1f64969.png)
매크로 메뉴에 있는 새로고침 주기를 변경하시면 됩니다.
